.github/workflows/build-system-test.yml

```diff
@@ -6,8 +6,9 @@ permissions:
   id-token: write # This is required for aws-actions/configure-aws-credentials
 
 on:
-  pull_request_target:
-   types: [opened, synchronize, labeled]
+  pull_request:
+    branches: [next-release/merge-main-to-migration]
+    types: [opened, synchronize, labeled]
```

Comment:
Since this is not in a new file and will need to reverted can you please add a comment using the `@todo-migration` flag

Reply:
Not sure what you mean since I'm not planning on merging this PR.

Reply:
That wasn't initially clear so no worries